rosidl_generator_py/resource/msg.py.template

```diff
@@ -0,0 +1,27 @@
+# Generated by rosidl_generator_py
+
+class @(spec.base_type.type):
+@[if len(spec.fields) > 0]@
+    __slots__ = ['@("', '".join([field.name for field in spec.fields]))']
```

Comment:
Since a message might contain many fields should we generate the code like this:

```
__slots__ = [
    'field_1',
    'field_2',
]
```

This would also avoid needing special handling for message without fields.